src/test/scala/uk/gov/hmrc/stubs/TestDataFactory.scala

```diff
@@ -71,6 +98,21 @@ object TestDataFactory {
     order = First
   )
 
+  def validSeniorAccountingOfficer(
+    fullName: String = Defaults.contactName
+  ): SeniorAccountingOfficer = SeniorAccountingOfficer(
+    fullName = fullName,
+    email = emailFor(fullName),
+    accountingPeriod = validSeniorAccountingOfficerPeriod()
```

Comment:
You could probably just remove `validSeniorAccountingOfficerPeriod()`, and do this:

```suggestion
    accountingPeriod = validAccountingPeriod().copy(dueDate = None)
```

Reply:
validSeniorAccountingOfficerPeriod removed and amended based on code provided